app/helpers/application_helper.rb

```diff
@@ -1,2 +1,9 @@
+# frozen_string_literal: true
+
 module ApplicationHelper
+  def neos_libraries
```

Comment:
I'd probably move this to the controller where you are using it? Typically you don't want SQL queries within the helper or views. That's the controller job to quarterback that work. 

E.g its being called in catalog/_home_text partial? I assume thats maybe the index page? So you can set a `@neos_libraries` instance variable inside the index action etc.